src/vma/dev/ring_simple.cpp

```diff
@@ -1017,7 +1014,7 @@ bool ring_simple::rx_process_buffer(mem_buf_desc_t* p_rx_wc_buf_desc, void* pv_f
 			NIPQUAD(p_ip_h->daddr), NIPQUAD(p_ip_h->saddr),
 			(sz_data > ip_tot_len ? ip_tot_len : sz_data), n_frag_offset, ntohs(p_ip_h->id),
 			iphdr_protocol_type_to_str(p_ip_h->protocol), p_ip_h->protocol,
-			NIPQUAD(p_rx_wc_buf_desc->rx.dst.sin_addr.s_addr));
+			NIPQUAD(p_rx_wc_buf_desc->rx.dst_addr));
```

Comment:
do you think it meets `local if` value? Probably the initial idea is to provide `rx.udp.local_if`

Reply:
I agree, but this will require additional changes (mainly because local_if is not updated for tcp).
i dont sure it worth it...